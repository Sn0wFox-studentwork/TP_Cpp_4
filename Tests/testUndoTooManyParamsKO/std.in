S s1 0 0 1 1
UNDO 5
EXIT
