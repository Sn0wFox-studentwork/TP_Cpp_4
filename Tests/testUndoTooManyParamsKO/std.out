ERR
#Must not have any parameters